app/models/post.rb

```diff
@@ -1,5 +1,6 @@
 class Post < ActiveRecord::Base
+  attr_accessor :image
```

Comment:
Post has a database field for `image`, so an `attr_accessor` here is unnecessary.